### PR TITLE
Add new test case for `Grafana` and `Node.js`

### PR DESCRIPTION
**Pull request by GabrielEValenzuela**
|Related issue|
|---|
|#25227|

# Objetive
This PR introduces a new test case to our Quality Assurance (QA) suite specifically designed to assess the translation functionality of the scanner against false negatives. The goal is to ensure the correct behavior of the scanner, enhancing our software's confiability for users.

## Evidence

<details><summary>Show logs</summary>

- Test output

```console

python -m pytest -k "run_process_and_monitor_log23"  wazuh_modules/vulnerability_scanner/qa/
============================================================ test session starts =============================================================
platform linux -- Python 3.11.9, pytest-7.2.2, pluggy-1.5.0
rootdir: /home/gvalenzuela/Documents/Work/wazuh/src
collected 30 items / 29 deselected / 1 selected                                                                                              

wazuh_modules/vulnerability_scanner/qa/test_efficacy_log.py .                                                                          [100%]

====================================================== 1 passed, 29 deselected in 2.34s ======================================================

```

</details>

### CODE REVIEW

**Comment by sebasfalcone** [comment]
## Failing tests
Because the tests were performed with the latest content (with offsets), some of the tests are failing

In the PR description we can found a local run with the latest offsets

**Comment by sebasfalcone** [comment]
## Failing tests
Because the tests were performed with the latest content (with offsets), some of the tests are failing

In the PR description we can found a local run with the latest offsets

**Comment by pereyra-m** [comment]
## Update

The tests were update considering the last changes in the content:
- Some CVEs were expected for `grafana` when in fact the affected package is `grafana-enterprise`. These entries were removed.
- Some non-vulnerable log entries were expected but now that the sanitizations were applied, those candidates aren't considered any more. Those entries were also removed.
- The changes from #23935 were used to replace the folder 034 because it was testing the same translation than test 033 but using Ubuntu packages over CentOS. 
- The feed in the test 033 was updated, because `grafana` is scanned with the NVD and `node.js` with Canonical.

**Review by sebasfalcone — CHANGES_REQUESTED** [review]
@pereyra-m Please rebase into master

**Comment by pereyra-m** [comment]
The tests are failing because I've added extra commits that:
- Fix a communication error that prevented the packages scanning (`Socket error request() got an unexpected keyword argument 'chunked'`)
- Check if we have no expected data in the test

**Comment by pereyra-m** [comment]
I roll back the extra changes:
- The version downgrade: `requests==2.28.1`
- The verification for empty expected files

<details><summary>Details</summary>
<p>

```c++
# Validate we have expected files
        if len(expected_json_files) == 0:
            process.terminate()
            LOGGER.error("The test can't continue because there isn't expected files")
            pytest.fail()
            
            ...
            
              # Validate we have expected data
        if quantity_expected_lines == 0:
            process.terminate()
            LOGGER.error("The test can't continue because there isn't expected data")
            pytest.fail()
```

</p>
</details>

File: src/engine/source/vdscanner/qa/test_false_negative_data/033/input_001.json
Comment (Dwordcito) [inline src/engine/source/vdscanner/qa/test_false_negative_data/033/input_001.json:1]: Package list scanning with empty array of packages ? Probably this input is not needed.

File: src/engine/source/vdscanner/qa/test_false_negative_data/034/input_001.json
Comment (Dwordcito) [inline src/engine/source/vdscanner/qa/test_false_negative_data/034/input_001.json:1]: Package list scanning with empty array of packages ? Probably this input is not needed.

File: src/engine/source/vdscanner/qa/test_false_negative_data/035/input_001.json
Comment (Dwordcito) [inline src/engine/source/vdscanner/qa/test_false_negative_data/035/input_001.json:1]: Package list scanning with empty array of packages ? Probably this input is not needed.

**Review by Dwordcito — CHANGES_REQUESTED** [review]

File: src/engine/source/vdscanner/qa/test_false_negative_data/033/input_001.json
Comment (GabrielEValenzuela) [reply src/engine/source/vdscanner/qa/test_false_negative_data/033/input_001.json:1]: Done!

File: src/engine/source/vdscanner/qa/test_false_negative_data/034/input_001.json
Comment (GabrielEValenzuela) [reply src/engine/source/vdscanner/qa/test_false_negative_data/034/input_001.json:1]: Done!

File: src/engine/source/vdscanner/qa/test_false_negative_data/035/input_001.json
Comment (GabrielEValenzuela) [reply src/engine/source/vdscanner/qa/test_false_negative_data/035/input_001.json:1]: Done!

**Review by sebasfalcone — APPROVED** [review]

**Review by Dwordcito — APPROVED** [review]

**Pull request merged**